seq2seq/__init__.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+logger = logging.getLogger('seq2seq')
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
```

Comment:
I think that should be there , makes it easier to read what module is executing. But I do agree that it looks a bit ugly in the console.